app/Controllers/feedController.php

```diff
@@ -722,13 +720,18 @@ public function actualizeAction(): int {
 		return $updated_feeds;
 	}
 
+	/**
+	 * @throws Minz_ConfigurationNamespaceException
+	 * @throws JsonException
```

Comment:
This line (several instances) looks wrong. Fixed in https://github.com/FreshRSS/FreshRSS/pull/6037